doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
+```
+e.g.
+```
+dotnet new dataplane --libraryName DeviceUpdate --swagger https://github.com/Azure/azure-rest-api-specs/blob/23dc68e5b20a0e49dd3443a4ab177d9f2fcc4c2b/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2021-06-01-preview/deviceupdate.json --securityScopes https://api.adu.microsoft.com/.default --force
+```
 
-#### 1. Set the Swagger File
+#### 3. update the solution file if needed
+Run 'dotnet sln' as following to update the projects in the solution file:
 
-Open autorest.md in your `Azure.<group>.<service>` project. In the input-file definition, replace the file path with your swagger link, for example: [WebPubSub swagger link](https://github.com/Azure/azure-rest-api-specs/blob/39c7d63c21b9a29efe3907d9b949d1c77b021907/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json). Update a namespace definition to set the namespace C# generator will use when creating your types. Update the AAD token credential(for AAD token, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/src/autorest.md#autorest-configuration) example) or Azure key credential (for Azure Key credential, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/src/autorest.md#swagger-sources) example). Save your changes.
+```
+dotnet sln remove src\Azure.<group>.<service>.csproj
+dotnet sln add src\Azure.<group>.<service>.csproj
+dotnet sln remove tests\Azure.<group>.<service>.Tests.csproj
+dotnet sln add tests\Azure.<group>.<service>.Tests.csproj
+```
 
-Your `autorest.md` file should now look like:
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
 
-````md
-# `Azure.<group>.<service>` Code Generation
+### Generate client library
 
-Run `dotnet build /t:GenerateCode` to generate code.
+In this section, we'll create a generated API layer built on Azure Core.
 
-### AutoRest Configuration
-> see https://aka.ms/autorest
+From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.csproj`. Run the following commands:
 
-``` yaml
-input-file:
-- <swagger-file-link>
-namespace: <namespace>
-public-clients: true
-data-plane: true
-security: AADToken
-security-scopes: <AAD-token-scope>
 ```
-````
+sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet build /t:GenerateCode
+```
 
-**Learn more:** see the [README.md](https://github.com/Azure/autorest.csharp/blob/feature/v3/readme.md) to learn more about Azure SDK Code Geneneration.
+After you run the GenerateCode command, you should find a **Generated** folder in your project. Inside the Generated folder you'll find the ServiceClient and ServiceClientOptions which you can use to interact with the service.
 
```

Comment:
Somewhere, and here seems like as good a place as any, we need to tell teams how to add convenience / "grow up" code.  There are certainly right and wrong ways to do this, so we need to give service teams some guidance on this. We could just give the basics here and then point to another document with full details, since most service teams won't bother to add convenience code, but we need the information here in case they do.

Update: I see now something at the very end of the file: **Learn more**.  I think that is way too late and it needs to point to specific material on adding convenience/grow up -- service teams are not going to wade through the whole readme to find this information.

Reply:
I updated in customize chapter. Please help to review

Reply:
@mikekistler, I think we should wait until after the "starter package" section to provide details regarding customization.  To make sure we get it in sooner, we could summarize the process in the introduction, mention in the starter package that you can do customization after the starter package is generated and checked-in (and passing CI), and then provide details in the customization section.

Reply:
Good point that we probably want to get the generated code checked in and passing CI before adding any extra code.